payments/swish/after-payment.md

```diff
@@ -280,17 +140,17 @@ Content-Type: application/json
 }
 ```
 
-#### Create Sales transaction
+### Create Sales transaction
 
-In e-commerce the consumer/end-user's `msisdn`(mobile number) is required.
+In browser based solutions the payers `msisdn`(mobile number) is required.
 This is managed either by sending a `POST` request as seen below,
 or by redirecting the end-user to the hosted payment page.
-The `msisdn` is only required for e-commerce. In the m-commerce flow,
-the consumer uses the device that hosts the Swish app to manage the purchase,
-making `msisdn` unneccessary.
+The `msisdn` is only required for browser based solutions.
+In-app solutions and flow, the consumer uses the device that hosts the Swish app
+to manage the purchase, making `msisdn` optional.
 
 {:.code-header}
-**e-commerce Request**
+**Browser-based Request**
 
 ```http
 POST /psp/swish/payments/<paymentId>/sales HTTP/1.1
```

Comment:
Please use real UUID values (preferably the same everywhere) instead of `<payment.id>` and `<paymentId>`.

```suggestion
POST /psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales HTTP/1.1
```